ta/trusted_key/include/trusted_key.h

```diff
@@ -0,0 +1,47 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2019-20, Linaro Limited
+ */
+
+#ifndef TRUSTED_KEY_H
+#define TRUSTED_KEY_H
+
+#define TRUSTED_KEY_UUID { 0xf04a0fe7, 0x1f5d, 0x4b9b, \
+		{ 0xab, 0xf7, 0x61, 0x9b, 0x85, 0xb4, 0xce, 0x8c } }
+
+/*
+ * Get random data for symmetric key
+ *
+ * [out]     memref[0]        Random data
+ *
+ * Result:
+ * TEE_SUCCESS - Invoke command success
+ * TEE_ERROR_BAD_PARAMETERS - Incorrect input param
```

Comment:
Minor: given the obviousness (?) of this "Result" part, I would not add it.
If it really makes sense to document that only those two codes may be returned, then: `Result: TEE_SUCCESS, TEE_ERROR_BAD_PARAMETERS` would be enough.

Reply:
Okay will drop this "Result" part.